src/rust/ide/src/view/node_editor.rs

```diff
@@ -801,6 +820,16 @@ impl GraphEditorIntegratedWithControllerModel {
         self.node_views.borrow().get_by_right(&displayed_id).cloned().ok_or(err)
     }
 
+    fn get_controller_node_ids
```

Comment:
i'm not sure - in rust getters generally don't start with word get..., but there are many places in our codebase where it happens - this can be ignored, or you can change it, for me it's ok, but i remembr Wojtek telling me the same ... 🤖

Reply:
I could argue that this is not a "getter" (as in just accessing some directly stored value) — it performs full lookup. But TBH I don't really have a strong opinion one way or another here. 
Anyway, it is implemented on top of `get_controller_node_id`, so for consistency let's keep it.